docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -36,130 +41,78 @@ log data has the following data retention guidelines:
+tier.
++
+[role="screenshot"]
+image::images/ilm/tutorial-ilm-ess-add-warm-data-tier.png[Add a warm data tier to your deployment]
 
-** Self-managed cluster: 
-Add node attributes as described for {ref}/shard-allocation-filtering.html[shard allocation filtering].
+** Self-managed cluster:
+Assign `data_hot` and `data_warm` roles to nodes as described in
+<<data-tiers>>.
 +
-For example, you can set this in your `elasticsearch.yml` for each data node:
+For example, include the `data_warm` node role in the `elasticsearch.yml` file
+of each node in the warm tier:
 +
 [source,yaml]
---------------------------------------------------------------------------------
-node.attr.data: "warm"
---------------------------------------------------------------------------------
+----
+node.roles: [ "data_warm" ]
+----
 
-* A server with {filebeat} installed and configured to send logs to the `elasticsearch`
-output as described in the {filebeat-ref}/filebeat-installation-configuration.html[{filebeat} quick start].
+* A host with the {agent} installed and configured to send logs to your {es}
+cluster.
 
 [discrete]
-[[example-using-index-lifecycle-policy-view-fb-ilm-policy]]
-==== View the {filebeat} {ilm-init} policy
+[[example-using-index-lifecycle-policy-view-ilm-policy]]
+==== View the policy
 
-{filebeat} includes a default {ilm-init} policy that enables rollover. {ilm-init}
-is enabled automatically if you’re using the default `filebeat.yml` and index template.
+The {agent} uses data streams with an index pattern of `logs-*-*` to store log
+monitoring data. {es}'s built-in `logs` {ilm-init} policy automatically manages
+backing indices for these data streams.
 
-To view the default policy in {kib}:
+To view the `logs` policy in {kib}:
 
-. Go to Management and select *Index Lifecycle Policies*.
-. Search for _filebeat_
-. Select the _filebeat-version_ policy.
+. Open the menu and go to **Stack Management > Index Lifecycle Policies**.
+. Select the `logs` policy.
 
-This policy initiates the rollover action when the index size reaches 50GB or
-becomes 30 days old.
+The `logs` policy uses our recommended rollover defaults. These defaults
+initiate a rollover when the current write index reaches 50GB or becomes 30 days
+old.
 
-[role="screenshot"]
-image::images/ilm/tutorial-ilm-hotphaserollover-default.png["Default policy"]
+To confirm these rollover settings, click **Settings** for the hot phase. Then
+disable **Use recommended defaults** to display the rollover settings.
 
+[role="screenshot"]
+image::images/ilm/tutorial-ilm-hotphaserollover-default.png[View rollover defaults]
 
 [discrete]
+[[ilm-ex-modify-policy]]
 ==== Modify the policy
 
-The default policy is enough to prevent the creation of many tiny daily indices.
-You can modify the policy to meet more complex requirements.
+The `logs` policy's default settings are enough to prevent the creation of many
+tiny daily indices. You can modify the policy to meet more complex requirements.
```

Comment:
```suggestion
The default `logs` policy is designed to prevent the creation of many
tiny daily indices. You can modify the policy to meet your performance requirements and manage resource usage.
```